### ISSUE TITLE
Where are the Elixir communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/elixir#404]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Elixir supported by one or more large organizations?
* Does Elixir have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Elixir? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Elixir? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Elixir? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Elixir?
* Are there popular newsletters for Elixir?
* Is Elixir taught at programming bootcamps? (If so, what are some examples?)
* Is Elixir taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Elixir?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by pdgonzalez872** [comment]
Hi @kytrinyx, thanks for your work with Exercism!

I'll try to tackle some of the questions you asked. If you'd like me to research more about a certain answer, please ask and I will do.

> - Is Elixir supported by one or more large organizations?

Yes, Here is a list of companies that sponsor Elixir http://sponsors.elixirforum.com/. Good question.

> - Does Elixir have an official community manager?

Another good question. There is a [forum](https://elixirforum.com/) that I'm sure someone is responsible for. It is **very** active and friendly. Overall the Elixir community is extremely helpful and friendly.

> - Do you know of specific communities (online or offline) that are enthusiastic about Elixir? (Chat communities, forums, meetups, student clubs, etc)

Yes: [forum](https://elixirforum.com/), Slack [channel](https://elixir-slackin.herokuapp.com/), lots of [meetups](https://www.meetup.com/topics/elixir/all/)

> - Are there popular conferences for Elixir? (If so, what are some examples?)

Yes. Here is the list with their details: https://github.com/elixir-lang/elixir/wiki/Conferences

> - Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Elixir? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

Good question. I'll research and get back to you on this.

> - Are there specific groups or programs dedicated to mentoring people in Elixir?

The Slack channel has been extremely helpful. I don't think there are any official mentoring groups specific for Elixir. That'd be awesome though.

> - Are there popular newsletters for Elixir?

Yes, https://github.com/h4cc/awesome-elixir#newsletters 

> - Is Elixir taught at programming bootcamps? (If so, what are some examples?)

I'm trying to at least give an option to the folks at [Code Platoon](https://www.codeplatoon.org/) to ask questions about Elixir. I'm building https://github.com/pdgonzalez872/elixir_track and will make myself available to help once the next cohort finishes. They do not offer Elixir, but I personally would like to make it an option to anyone that goes through CodePlatoon and wants to look into the language.

> - Is Elixir taught at universities? (If so, what are some examples?)

I'm not sure how often it happens, [Dave Thomas](https://www.smu.edu/Lyle/Departments/CSE/People/AdjunctFaculty) has taught a course at SMU using Elixir as the main language.

Hope this helps. I'll post this in the forum and maybe some folks can help me with the ones I didn't know.

**Comment by kytrinyx** [comment]
This is incredibly helpful, thank you!

**Comment by pdgonzalez872** [comment]
You got it. I posted in the Forum and also the Elixir general channel. Hopefully we will fill out all of the questions.

Exercism + Elixir  ->  :heart:

**Comment by elbow-jason** [comment]
Conferences: https://github.com/elixir-lang/elixir/wiki/Conferences

**Comment by sleepiejohn** [comment]
Just a note @pdgonzalez872.
On the question **Is Elixir supported by one or more large organizations?** http://sponsors.elixirforum.com/ is a list of sponsors of the forum and community, i.e. they help with bills of server, make giveways, books etc...

On the development side, Elixir is supported by [Plataformatec](http://plataformatec.com.br/)

On consultancy side, there is [Plataformatec](http://plataformatec.com.br/) again and [Erlang Solutions](https://www.erlang-solutions.com/) afaik

**Comment by makefunstuff** [comment]
>Are there popular conferences for Elixir? (If so, what are some examples?)

https://codesync.global/ is one of the best conferences I ever been. Various erlang/elixir topics for different levels. Highly recommend!

**Comment by pdgonzalez872** [comment]
Here is the forum thread: https://elixirforum.com/t/help-with-answering-questions-about-elixir/14512, a couple of answers there as well.

**Comment by elbow-jason** [comment]
On the consultancy side there is also [DockYard](https://dockyard.com/).

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. (Should you come across any other good sources, feel free to add them here, I'll be coming back to this). ❤️ 💛 💚 💙 💜 🖤 Thanks!
